### PR TITLE
Removed HgBitbucketDriver support

### PR DESCRIPTION
**Pull request by raphaeldealmeida**
Removed HgBitbucketDriver support and changed VCS bitbucketDriver from 'git-bitbucket' to 'bitbucket'
This BREAKS backward compatibility.

Original issue #9856

### CODE REVIEW

**Comment by stof** [comment]
Removing `git-bitbucket` is a no-go to me due to the BC break. We can *add* `bitbucket` as an alias, but we cannot remove `git-bitbucket`

**Comment by raphaeldealmeida** [comment]
> Removing `git-bitbucket` is a no-go to me due to the BC break. We can _add_ `bitbucket` as an alias, but we cannot remove `git-bitbucket`

I created an alias from git-bitbucket to bitbucket reducing BC break.
Please review my PR

File: doc/05-repositories.md [hunk 1]
```diff
@@ -412,6 +412,7 @@ attempt to use github's zip files.
 Please note:
 * **To let Composer choose which driver to use** the repository type needs to be defined as "vcs"
 * **If you already used a private repository**, this means Composer should have cloned it in cache. If you want to install the same package with drivers, remember to launch the command `composer clearcache` followed by the command `composer update` to update Composer cache and install the package from dist.
+* VCS driver `git-bitbucket` is depreceted in favor of `bitbucket`
```
Comment (Seldaek) [inline doc/05-repositories.md:415]:
```suggestion
* VCS driver `git-bitbucket` is deprecated in favor of `bitbucket`
```

**Review by Seldaek — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**